cmd/basin/publication.go

```diff
@@ -392,7 +392,6 @@ func newPublicationDealsCommand() *cli.Command {
 				Name:        "latest",
 				Usage:       "The latest N deals to fetch",
 				Destination: &latest,
-				Value:       10,
```

Comment:
removed the default value for `latest` because of [this IF ](https://github.com/tablelandnetwork/basin-cli/pull/13/files#diff-73777451ff3bdcfcd6ee6970408801e8731a6368a183dfca6d1e1b7df2195fc7R422). if `latest` has a default value, `limit` and `offset` will never be used